clusters/hosted-mgmt/hive/pools/pipelines/pipelines-ocp-4-15-0-amd64-openstack-clusterpool.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: psi-pipelines
+    owner: pipelines
+    product: ocp
+    version: "4.15"
+    version_lower: 4.15.0-0
+    version_upper: 4.16.0-0
+  name: pipelines-ocp-4-15-0-amd64-openstack
+  namespace: pipelines-cluster-pool
+spec:
+  baseDomain: psi.ospqa.com
+  hibernationConfig:
+    resumeTimeout: 20m0s
+  imageSetRef:
+    name: ocp-release-4.15.12-x86-64-for-4.15.0-0-to-4.16.0-0
+  installAttemptsLimit: 1
+  installConfigSecretTemplateRef:
+    name: pipelines-install-config
+  labels:
+    tp.openshift.io/owner: pipelines
+  platform:
+    openstack:
+      cloud: psi-pipelines
+      credentialsSecretRef:
+        name: pipelines-openstack-credentials
+  size: 0
+  skipMachinePools: true
+status:
```

Comment:
same as above

Reply:
Done, thanks for the review @deepsm007 🙂